build/configure/src/bundle.rs

```diff
@@ -250,10 +252,17 @@ fn install_anki_wheels(build: &mut Build) -> Result<()> {
 }
 
 fn build_pyoxidizer(build: &mut Build) -> Result<()> {
+    let mut repo_type = false;
```

Comment:
I'd rename this to offline_build, then use

let offline_build = env::var("OFFLINE_BUILD").is_ok();

Reply:
Thanks for the suggestion should be fixed.